LEContextMenuHandler/Lang/fr.xml

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 
 <Strings>
-  <RunDefault>Executer avec le profil de cette application</RunDefault>
-  <Submenu>Locale Emulator</Submenu>
+  <RunDefault>Éxecuter avec le profil de cette application</RunDefault>
+  <Submenu>Émulateur local</Submenu>
```

Comment:
Locale Emulator is the name of the application, it is localized in some lang files but not in others. I think it is better unify this translation string. @regomne